cmd/ui/src/views/Explore/EntityInfo/EntityInfoPanel.tsx

```diff
@@ -82,7 +59,15 @@ const EntityInfoPanel: React.FC<EntityInfoPanelProps> = ({ selectedNode, sx }) =
                 style={{
                     display: expanded ? 'initial' : 'none',
                 }}>
-                <EntityInfoContent id={selectedNode?.id} nodeType={selectedNode?.type} />
+                {selectedNode ? (
+                    <EntityInfoContent
+                        id={selectedNode.id}
+                        nodeType={selectedNode.type}
+                        databaseId={selectedNode.graphId}
```

Comment:
is graphId something neo4j is setting per node? Such as the `ID(node)` in cypher? Hopefully it is because that would mean if there was a node created outside our collectors that it would still have an id

Reply:
amazin, thank you! That would be a good improvement in the future!